packages/devextreme/js/__internal/core/ai_integration/core/ai_integration.test.ts

```diff
@@ -9,18 +9,66 @@ import {
 import type {
   AIProvider,
   RequestCallbacks,
-  TranslateCommandParams,
 } from '@js/common/ai-integration';
-import { TranslateCommand } from '@ts/core/ai_integration/commands/translate';
+import {
+  ChangeStyleCommand,
+  ChangeToneCommand,
+  ExecuteCommand,
+  ExpandCommand,
+  ProofreadCommand,
+  ShortenCommand,
+  SummarizeCommand,
+  TranslateCommand,
+} from '@ts/core/ai_integration/commands';
 import { AIIntegration, CommandNames } from '@ts/core/ai_integration/core/ai_integration';
 import { PromptManager } from '@ts/core/ai_integration/core/prompt_manager';
 import { RequestManager } from '@ts/core/ai_integration/core/request_manager';
 import { Provider } from '@ts/core/ai_integration/test_utils/provider_mock';
 
-describe('AI', () => {
-  const params: TranslateCommandParams = { text: 'text for translation', lang: 'French' };
-  const params2: TranslateCommandParams = { text: 'text for translation 2', lang: 'Spanish' };
-
+const COMMANDS = {
+  [CommandNames.ChangeStyle]: {
+    command: ChangeStyleCommand,
+    params: { text: 'text to style change' },
+    params2: { text: 'text to style change 2' },
```

Comment:
[excess code?]
Hm, I don't see `params2` is used somewhere

Reply:
Used in `it(`should reuse the same command instance for multiple ${commandName} calls`, () => {`